proto/umee/leverage/v1/query.proto

```diff
@@ -335,19 +151,84 @@ message QueryMarketSummaryResponse {
   string available_withdraw = 17 [
     (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int",
-    (gogoproto.nullable)   = false,
-    (gogoproto.jsontag)    = "available_withdraw"
+    (gogoproto.nullable)   = false
   ];
   // Available Collateralize is the maximum additional amount of uTokens than can be collateralized based on current liquidity and system safety limits. It can also be calculated by (maximum_collateral, - collateral). It is denominated in uTokens, so both uToken exchange rate and exponent must be applied to convert to symbol denom. A negative availability means safety limits have been exceeded and additional collateral cannot be created until more liquidity is present.
   string available_collateralize = 18 [
     (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int",
-    (gogoproto.nullable)   = false,
-    (gogoproto.jsontag)    = "available_collateralize"
+    (gogoproto.nullable)   = false
   ];
-}
\ No newline at end of file
+}
+
+// QueryAccountSummary defines the request structure for the AccountSummary gRPC service handler.
+message QueryAccountSummary {
+  string address = 1;
+}
+
+// QueryAccountSummaryResponse defines the response structure for the AccountSummary gRPC service handler.
+message QueryAccountSummaryResponse {
+  // Supplied contains all tokens the account has supplied, including interest earned. It is denominated in base tokens, so exponent from each coin's registered_tokens entry must be applied to convert to symbol denom.
+  repeated cosmos.base.v1beta1.Coin supplied = 1 [
+    (gogoproto.nullable)     = false,
+    (gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.Coins"
+  ];
+  // Collateral contains all uTokens the account has collateralized. It is denominated in uTokens, so both exponent and uToken exchange rate from each coin's market_summary must be applied to convert to base token symbol denom.
+  repeated cosmos.base.v1beta1.Coin collateral = 2 [
+    (gogoproto.nullable)     = false,
+    (gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.Coins"
+  ];
+  // Borrowed contains all tokens the account has borrowed, including interest owed. It is denominated in base tokens, so exponent from each coin's registered_tokens entry must be applied to convert to symbol denom.
+  repeated cosmos.base.v1beta1.Coin borrowed = 3 [
+    (gogoproto.nullable)     = false,
+    (gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.Coins"
+  ];
+}
+
+// QueryAccountHealth defines the request structure for the AccountHealth gRPC service handler.
+message QueryAccountHealth {
+  string address = 1;
+}
+
+// QueryAccountHealthResponse defines the response structure for the AccountHealth gRPC service handler.
+message QueryAccountHealthResponse {
```

Comment:
With `Health` I would expect some kind of metric, maybe 0-100%. Here we are returning aggregated balances. 

Let's change Health in the query, request and response names to something else. Maybe:
1. we can rename `QueryAccountSummary` to `QueryAccountBalances` and use the former name here (renaming `QueryAccountHealth` to `QueryAccountSummary`)?

Reply:
Good idea - that solves the naming ambiguity